manifests/00-crd-extension-console-notification-banner.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  # TODO: consolenotificationbanners.console.openshift.io
+  # will we ever have consolenotifications that act like
+  # traditional notifications?
+  name: consolenotifications.console.openshift.io
+  annotations:
+    displayName: ConsoleNotification
```

Comment:
Really think this should be `ConsoleNotificationBanner`.  This isn't how `Notification` is typically understood.

Reply:
ConsoleNotification.